identity/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "chrome.identity example",
```

Comment:
@wbamberg has standardized on using browser instead of chrome, so that should apply for all the APIs in this.